sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -356,6 +374,85 @@ def wrapper(*args, **kwargs):
   return wrap
 
 
+BEAM_SPECIFIC = "Differences from pandas"
+
+SECTION_ORDER = [
+    'Parameters',
+    'Returns',
+    'Raises',
+    BEAM_SPECIFIC,
+    'See Also',
+    'Notes',
+    'Examples'
+]
+
+EXAMPLES_DISCLAIMER = (
+    "**NOTE:** These examples are pulled directly from the pandas documentation "
+    "for convenience. The Beam DataFrame API will look different because it is "
```

Comment:
"The Beam DataFrame API" -> "Usage of the Beam DataFrame API"

Reply:
Done